src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -31,115 +42,157 @@ object Conseil
     with ConseilOutput {
 
   loadApplicationConfiguration(args) match {
-    case Right((server, platforms, securityApi, verbose, metadataOverrides, nautilusCloud)) =>
+    case Left(errors) =>
```

Comment:
Instead of writing this out as a code block could we make the running of Conseil a one-line composition of multiple functions? This will make the code a lot clearer to read.

Reply:
Do you mind if I join this proposal with #581 to have a more uniform definition for both the main application entrypoints?
@vishakh

Reply:
As discussed, this will be done as a separate effort.